statsmodels/tsa/vector_ar/util.py

```diff
@@ -215,10 +215,11 @@ def varsim(coefs, intercept, sig_u, steps=100, initvalues=None, seed=None):
         observations to start the autoregressive process.
         If offset is not None, then exog of the model are used if they were
         provided in the model
+    initvalues : None or ndarray (nlags, neqs)
```

Comment:
Type should be 

ndarray, optional

Reply:
After you make the other changes, the shape should move to the description.  It should be either (neqs,) or (nlags, neqs)

Reply:
Sorry, should be array_like, optional

then you need to use asarray below, or better yet the function `array_like`

Reply:
Also, please rename to `initial_values` for consistency with below.